app/models/poll.rb

```diff
@@ -49,11 +49,17 @@ def loaded_options
   end
 
   def possibly_stale?
-    remote? && last_fetched_before_expiration? && time_passed_since_last_fetch?
+    remote? && !final? && time_passed_since_last_fetch?
+  end
+
+  def final?
```

Comment:
This may be overthinking, but naming-wise here, I sort of interpret `poll.final?` as "is this the last poll in some list?" ... thoughts on `poll.finalized?`, `poll.completed?`, `poll.closed?`, etc...?